src-worker/doser.ts

```diff
@@ -31,16 +31,28 @@ export class Doser {
   private working: boolean
   private workers: number
   private eventSource: EventEmitter
+  private verboseError: boolean;
 
   private workerActive: Array<boolean>
 
-  constructor (onlyProxy: boolean, workers: number) {
+  constructor (onlyProxy: boolean, workers: number, verboseError = true) {
     this.onlyProxy = onlyProxy
     this.working = false
     this.workers = workers
     this.eventSource = new EventEmitter()
     this.workerActive = new Array<boolean>(256)
     this.workerActive.fill(false)
+    this.verboseError = verboseError;
+  }
+
+  private logError(message:string, cause: unknown){
+    console.log(message);
+
+    if (this.verboseError){
+      console.log(cause)
+    } else{
+      console.log((cause as AxiosError)?.message)
```

Comment:
You can guard TS with `isAxiosError`
example: https://github.com/axios/axios#typescript